### PR TITLE
Carnivore warns you about Scurvy fixing #70474

### LINKED ISSUE #70474: Carnivore develops Scurvy
### Describe the bug

My late-game Chimera mutant has developed Scurvy due to subsisting exclusively on smoked meat, despite the Carnivore mutation ostensibly making it so you don't have to worry about that.

This could have something to do with the character being a biokinetic (MindOverMatter). Maybe the Efficient System (BIOKIN_NEEDS) multiplier to vitamin absorbtion rates interacts weirdly with the negative vitamin_rate from Carnivore or something.

Alternatively it could be an interaction with another of my dozens of mutations.

### Attach save file

[Mindssachusetts-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/13779054/Mindssachusetts-trimmed.tar.gz)

### Steps to reproduce

Make a carnivore, set up an eating zone with nothing but smoked meat and see if they develop scurvy I guess.

### Expected behavior

Carnivores can live off meat without suffering from vitamin deficiencies.

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3803 (22H2)
- Game Version: 92a3704 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Mind Over Matter [mindovermatter],
    Bionic Slots [cbm_slots],
    Stats Through Kills [stats_through_kills],
    Massachusetts [MA]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Players shouldn't be caught offguard when they catch scurvy as carnivores.  Fixed #70474

#### Describe the solution
Adds an extra sentence in the carnivore trait.

### CODE REVIEW

**Comment by Fris0uman** [comment]
Isn't this a bit too long? Half the description is now bout scurvy.

**Comment by Maleclypse** [comment]
> Isn't this a bit too long? Half the description is now bout scurvy.

Fair I'll see if anyone can come up with a better description.

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -7689,7 +7689,7 @@
     "id": "CARNIVORE",
     "name": { "str": "Carnivore" },
     "points": -4,
-    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.",
+    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.  While your vitamin C needs are lowered, its still needed and can be found often in organ meat",
```
Comment (MNG-cataclysm) [inline data/json/mutations/mutations.json:7692]:
```suggestion
    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.  While your vitamin C needs are lowered, it's still needed and can be found often in organ meat.",
```

Alternatively, you could use `"Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.  While your vitamin C needs are lowered, it's still needed and can be found in organ meat."`

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -7689,7 +7689,7 @@
     "id": "CARNIVORE",
     "name": { "str": "Carnivore" },
     "points": -4,
-    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.",
+    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.  While your vitamin C needs are lowered, it's still needed and can be found often in organ meat.",
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:7692]:
```suggestion
    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.  While your vitamin C needs are lowered, it's still needed and can be found often in organ meat."
```

File: data/json/mutations/mutations.json [hunk 3]
```diff
@@ -7689,7 +7689,7 @@
     "id": "CARNIVORE",
     "name": { "str": "Carnivore" },
     "points": -4,
-    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.",
+    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.  While your vitamin C needs are lowered, it's still needed and can be found often in organ meat."
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:7692]:
```suggestion
    "description": "Your body's ability to digest fruits, vegetables, grains and nuts is severely hampered.  You cannot eat anything besides meat.  While your vitamin C needs are lowered, it's still needed and can be found often in organ meat.",
```

**Pull request merged**